src/test/java/com/laytonsmith/core/functions/ArrayHandlingTest.java

```diff
@@ -306,4 +305,20 @@ public void testStringSlice() throws Exception {
 	@Test public void testArrayUnique4() throws Exception {
 		assertEquals("{1, 1}", SRun("array_unique(array(1, '1', 1), true)", fakePlayer));
 	}
+	
+	@Test public void testArrayGetClone() throws Exception {
+		Run("@a = array(array(array('value'))); @b = @a[]; @b[0][0][0] = 'changedValue'; msg(@a[0][0][0]);", fakePlayer);
+		verify(fakePlayer).sendMessage("value");
```

Comment:
This should also msg @b[0][0][0] to insure that it is 'changedValue. Add the msg(@b...) to the Run, then add another verify(...) under line 311.